ui/cypress/support.js

```diff
@@ -1,23 +1,21 @@
-/* global Cypress, cy */
+/* global Cypress, cy, beforeEach */
 import '@testing-library/cypress/add-commands';
 import installLogsCollector from 'cypress-terminal-report/src/installLogsCollector.js';
 
 installLogsCollector();
 
-Cypress.Commands.add('login', (username = 'idtest0', password = '0000') => {
+beforeEach(() => {
   cy.visit('/');
+});
```

Comment:
Moving `cy.visit('/')` out of the `login` command to avoid an unnecessary page navigation in the new log out/in test.